airflow/include/tasks/extract/utils/html_utils.py

```diff
@@ -23,28 +24,72 @@ def is_valid_url(url):
     return bool(parsed.netloc) and bool(parsed.scheme)
 
 
-def exclude_path(url, exclude_docs=None):
+def fetch_page_content(url: str) -> str:
+    try:
+        response = requests.get(url)
+        response.raise_for_status()  # Raise an HTTPError for bad responses
+        return response.content
+    except requests.RequestException:
+        logger.info("Error fetching content for %s: %s", url, url)
+        return ""
+
+
+def exclude_url(url: str, exclude_literal: list[str]) -> bool:
```

Comment:
I'm a bit confused when I first read this name. Should we rename it to `is_excluded_url`?

Reply:
indeed, it was confusing to me too thanks for suggestions